### DISCUSSION TITLE
URL to nzyme 2.x  _deb_package

### DISCUSSION THREAD

**Discussion by sudo-joe** [nzymedefense/nzyme#1100] [General]
Hello

url_to_nzyme_deb_package

**Comment by sudo-joe** [comment]
Hello,

Where can debian packages of Nzyme 2.x be found?

https://docs.nzyme.org/getting-started/installation/ubuntu-server-2204/

Thank's in advance!

**Comment by lennartkoopmann** [comment]
Hi,

that section has a link to the downloads page: https://go.nzyme.org/downloads
